apps/api/src/app/access/access-filter.dto.ts

```diff
@@ -0,0 +1,40 @@
+import { DataSource } from '@prisma/client';
+import { Type } from 'class-transformer';
+import {
+  IsArray,
+  IsEnum,
+  IsOptional,
+  IsString,
+  ValidateNested
+} from 'class-validator';
+
+class HoldingFilterDto {
```

Comment:
Could you rename to `AssetProfileIdentifierDto` (inspired by `AssetProfileIdentifier` in `libs/common/src/lib/interfaces`) and move to `libs/common/src/lib/dto`